templates/content/social-media-links.html.twig

```diff
@@ -1,6 +1,9 @@
 <div class="social-media__items">
   {% set classes = ['social-media__item'] %}
-  {% for element in elements %}
+  {% for key, element in elements %}
+    {# Only email text is translated #}
+    {% set alternative_language = alternative_language and key != 'email' %}
```

Comment:
Why are you checking if `alternative_language` is `alternative_language`?

Reply:
Not really checking against it, re-setting it based on original value and value of `key`.
Pretty much like setting `x = x && y`

Reply:
After further discussion, this was refactored as old implementation would have broken if email would have been before any other element.